orbmain/src/main/java/com/sun/corba/ee/impl/util/Utility.java

```diff
@@ -302,10 +303,14 @@ static Class loadClassOfType(String className, String remoteCodebase,
                     "expected Type.");
             }
 
-            if (expectedTypeClassLoader != null) {
-                loadedClass = expectedTypeClassLoader.loadClass(className);
-            } else {
-                loadedClass = ORBClassLoader.loadClass(className);
+            try {
+                if (expectedTypeClassLoader != null) {
+                    loadedClass = expectedTypeClassLoader.loadClass(className);
+                } else {
+                    loadedClass = ORBClassLoader.loadClass(className);
+                }
+            } catch (Exception e) {
```

Comment:
When an exception occurs, will the process continue with the loadedClass being null without interrupting the process. will it be loaded elsewhere...?